pkg/mount/mounter_linux.go

```diff
@@ -4,15 +4,48 @@ import (
 	"syscall"
 )
 
-func mount(device, target, mType string, flag uintptr, data string) error {
-	if err := syscall.Mount(device, target, mType, flag, data); err != nil {
-		return err
+const (
+	// ptypes is the set propagation types.
+	ptypes = syscall.MS_SHARED | syscall.MS_PRIVATE | syscall.MS_SLAVE | syscall.MS_UNBINDABLE
+
+	// pflags is the full set valid flags for a change propagation call.
+	pflags = ptypes | syscall.MS_REC | syscall.MS_SILENT
+
+	// broflags is the combination of bind and read only
+	broflags = syscall.MS_BIND | syscall.MS_RDONLY
+)
+
+// isremount returns true if either device name or flags identify a remount request, false otherwise.
+func isremount(device string, flags uintptr) bool {
+	switch {
+	case flags&syscall.MS_REMOUNT != 0, device == "", device == "none":
```

Comment:
ok, unsure here. Why is device=="" or device="none" a remount? I can mount a tmpfs with these options. Why is presense of `MS_REMOUNT` not sufficient?